core/geotime/src/main/java/org/locationtech/geowave/core/geotime/store/query/filter/SpatialQueryFilter.java

```diff
@@ -379,7 +378,8 @@ public GeometryImage(final byte[] geometryBinary) {
 
     public synchronized void init() {
       if (preparedGeometry == null) {
-        preparedGeometry = FACTORY.create(GeometryUtils.geometryFromBinary(geometryBinary, null));
+        preparedGeometry =
+            PreparedGeometryFactory.prepare(GeometryUtils.geometryFromBinary(geometryBinary, null));
```

Comment:
I had originally moved the static factory to `GeometryUtils`, but I switched it to this method because it was only really being called once per query in `SpatialQueryFilter` and once per literal in the filter expressions, so it seemed silly to hold  a static instance around for that.